kb-survey/src/app/config/url.config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "baseURL":"http://localhost:3000",
```

Comment:
This should be part of the environment variable in environmnet files.

Reply:
ok, will do that